### PR TITLE
Update docs with react-query example

### PR DESCRIPTION
**Pull request by Navipro70**
Hi @mrousavy, I've met some troubles with react-query persist and MMKV connection and I think this docs will be usefull for people who are using react-query)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ A mocked MMKV instance is automatically used when testing with Jest, so you will
 * [Using MMKV with mobx-persist](./docs/WRAPPER_MOBXPERSIST.md)
 * [Using MMKV with zustand persist-middleware](./docs/WRAPPER_ZUSTAND_PERSIST_MIDDLEWARE.md)
 * [Using MMKV with jotai](./docs/WRAPPER_JOTAI.md)
+* [Using MMKV with react-query](./docs/WRAPPER_REACTQUERY.md)
```
Comment (mrousavy) [inline README.md:199]: Can you rename the file to `WRAPPER_REACT_QUERY.md` (`_` between REACT and QUERY)?

File: docs/WRAPPER_REACTQUERY.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# react-query wrapper
+
+If you want to use MMKV with [react-query](https://tanstack.com/query/v4/docs/plugins/createSyncStoragePersister), create the following `clientPersister` object:
```
Comment (mrousavy) [inline docs/WRAPPER_REACTQUERY.md:3]: Maybe also mention that you need to install the `@tanstack/query-sync-storage-persister` package?

File: docs/WRAPPER_REACTQUERY.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+```ts
+import { createSyncStoragePersister } from '@tanstack/query-sync-storage-persister'
+import { MMKV } from "react-native-mmkv"
+
+const storage = new MMKV();
+
+export const clientStorage = {
```
Comment (mrousavy) [inline docs/WRAPPER_REACTQUERY.md:11]: I don't think this should be exported, should it?

File: docs/WRAPPER_REACTQUERY.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  getItem: (key) => {
+    const value = storage.getString(key);
+    return value === undefined ? null : value;
+  },
+  removeItem: (key) => {
+    storage.delete(key);
+  },
+};
+
+const clientPersister = createSyncStoragePersister({ storage: clientPersistStorage });
```
Comment (mrousavy) [inline docs/WRAPPER_REACTQUERY.md:24]: Looks like this should be exported though!

**Review by mrousavy — CHANGES_REQUESTED** [review]

**Comment by mrousavy** [comment]
Thanks for your PR! Left some minor tweaks to perfect it but otherwise looks good!

**Comment by Navipro70** [comment]
@mrousavy 
Check please, I assume we can merge it)

**Comment by mrousavy** [comment]
Ah yep, looks good - thanks! :)

**Pull request merged**